code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -1463,6 +1463,10 @@ GLOBAL_LIST_EMPTY(features_by_species)
 		// Apply the damage to all body parts
 		humi.apply_damage(burn_damage, BURN, spread_damage = TRUE)
 
+	// For cold damage, we stop prior to husking
+	if(humi.getFireLoss() >= abs(HEALTH_THRESHOLD_DEAD * 1.75))
+		return
```

Comment:
I'd prefer it if this was way lower and only applied if you were dead.
I.e. you can't take more than 100 burn damage whilst dead but if you are alive, you take burn damage up until you die.

Reply:
okay